autobump-core/src/test/java/com/github/autobump/core/model/usecases/PullRequestUseCaseTest.java

```diff
@@ -0,0 +1,87 @@
+package com.github.autobump.core.model.usecases;
+
+import com.github.autobump.core.model.Bump;
+import com.github.autobump.core.model.CommitResult;
+import com.github.autobump.core.model.Dependency;
+import com.github.autobump.core.model.GitClient;
+import com.github.autobump.core.model.GitProvider;
+import com.github.autobump.core.model.PullRequest;
+import com.github.autobump.core.model.UrlHelper;
+import com.github.autobump.core.model.Version;
+import com.github.autobump.core.model.Workspace;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+class PullRequestUseCaseTest {
+    private GitProvider gitProvider;
+    private GitClient gitClient;
+    private UrlHelper urlHelper;
+    private Workspace workspace;
+    private Bump bump;
+    private URI uri;
+
+    @BeforeEach
+    void setUp() throws URISyntaxException {
+        createMocks();
+        setUpMocks();
+    }
+
+    private void createMocks() throws URISyntaxException {
```

Comment:
Mockito has an Extension specific for JUnit 5 that allows to easily create mocks.

```
@ExtendWith(MockitoExtension.class)
class PullRequestUseCaseTest {
    @Mock
    private GitProvider gitProvider;

    @Mock
    private GitClient gitClient;

    @InjectInto // this creates your class under test and automatically inject mocks.
    private ClassUnderTest classUnderTest;

    // alternatively you can construct the class under test yourself in a before callback and inject manually

    void test() {
        // ...
    }
```

Not necessary to fix in this PR, but I would create a new GitHub issue to clean this up in a new sprint.